corehq/apps/users/management/commands/gdpr_scrub_user.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+import xmltodict
+from django.core.management.base import BaseCommand
+from corehq.form_processor.interfaces.dbaccessors import FormAccessors
+
+
+class Command(BaseCommand):
+    help = "Scrubs the username from all forms associated with the given user"
+
+    def add_arguments(self, parser):
+        parser.add_argument('user_id')
+        parser.add_argument('domain')
+
+    def handle(self, user_id, domain, **options):
+        this_form_accessor = FormAccessors(domain=domain)
+        form_ids = this_form_accessor.get_form_ids_for_user(user_id)
+        new_username = "Redacted User (GDPR)"
+        for form_data in this_form_accessor.iter_forms(form_ids):
+            form_attachment_xml_new = self.parse_form_data(form_data, new_username)
+            this_form_accessor.modify_attachment_xml_and_metadata(form_data, form_attachment_xml_new)
+
+    @staticmethod
+    def parse_form_data(form_data, new_username):
```

Comment:
Nitpick: what do you think of renaming this to `update_form_data` since it doesn't just parse the form?

Reply:
Good idea, changed here:
https://github.com/dimagi/commcare-hq/pull/19979/commits/8d6837e0473a99499db893b8e79ac0d7e1afb62b